special-interest-groups/sig-expr/roles-and-organization-management.md

```diff
@@ -0,0 +1,60 @@
+# SIG Expr Roles and Organization Management
+
+## Join && Promotion
+
+## Join
+
+The minimal requirement for contributors to join the Expr SIG is to have at
+least the **Active Contributor** community role in the expression package.
```

Comment:
> in the expression package

Can we say "in the expression package subproject"?